doc/research/paf24/general/leaderboard_summary.md

```
@@ -0,0 +1,132 @@
+      ![image](https://leaderboard.carla.org/assets/images/TR19a.png)
+    - The ego-vehicle must slow down or brake to allow a parked vehicle exiting a parallel parking bay to cut in front.
+      ![image](https://leaderboard.carla.org/assets/images/TR12.png)
+    
+- ### **Parking:**
+    
+    - The ego-vehicle must exit a parallel parking bay into a flow of traffic.
+      ![image](https://leaderboard.carla.org/assets/images/TR11.png)
+
+## Available Sensors
+
+  - **[GNSS](https://carla.readthedocs.io/en/latest/ref_sensors/#gnss-sensor)**
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)